include/etcpal/cpp/thread.h

```diff
@@ -357,6 +361,19 @@ inline Thread& Thread::SetPlatformData(void* platform_data) noexcept
   return *this;
 }
 
+/// @brief Set the timeout for shutting down the thread
+///
+/// This function sets the timeout for destructing a thread. If the thread has not stopped within the timeout, it is
+/// forcefully quit.
```

Comment:
Based on the implementation of etcpal_thread_timed_join(), I'm not sure that this comment is true? Is the thread actually forcefully quit after failing to stop?

Reply:
Good point- its memory just gets removed in a dirty way. I can update the comment.

Reply:
I added a new function to forcefully stop the threads.